includes/class-wc-privacy.php

```diff
@@ -369,8 +369,55 @@ protected static function delete_inactive_accounts_query( $timestamp, $limit = 2
 				require_once ABSPATH . 'wp-admin/includes/user.php';
 			}
 
+			$builtin_post_types = get_post_types( array( '_builtin' => true ) );
```

Comment:
Now that we use `wp_delete_user()` to properly reassign posts seems like we don't need it anymore.

We could only query for pages, posts and products, so we don't list all content in the email, but we can place a message saying that more content got replaced.
This will for sure help a little in case the user was a lot of content, since currently we are fetching everything and doesn't sounds very important to list all in this email since you can also check in the back-end later.

Reply:
Hi @claudiosanches 

Let me check if I understand what you said... 

I should remove it:

```
$builtin_post_types = get_post_types( array( '_builtin' => true ) );
unset( $builtin_post_types['user_request'] ); // Exclude user_request from data migration.


/**
 * Filter post types that should be assign to an admin user prior user exclusion.
 *
 * @param array $post_types List of post types to assign to an admin. Defaults to WordPress builtin post types.
 */
$post_types_to_assign = apply_filters( 'woocommerce_delete_inactive_account_post_types_to_assign', array_values( $builtin_post_types ) );
```

And replace this:

```
$posts_made_prior = get_posts(
	array(
		'post_type'   => $post_types_to_assign,
		'author'      => $user_id,
		'numberposts' => -1,
	)
);
```

By something like this:

```
$posts_made_prior = get_posts(
	array(
		'post_type' => array( 'post', 'page', 'product' ),
		'author'      => $user_id,
		'numberposts' => -1,
	)
);
```

So in the e-mail message, we can say something like _"pages, posts, products, and others.."_.

**1)** Is this? 

**2)** Or we should keep the filter for the post types to assign? I have this doubt, although not make much sense to keep the filter since the **wp_delete_user()** will re-assign all the post types, right?

Awaiting your feedback to proceed! =)